Controllers/CmsController.cs

```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
                 return Unauthorized();
             }
         }
+        
+        /// <summary>
+        /// Gets the home page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/homepage")]
+        public async Task<IActionResult> HomePage(Guid id, bool draft = false)
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<HomePage>(id, HttpContext.User, draft);
+
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();
+            }
+        }
+        
+        /// <summary>
+        /// Gets the service page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/servicepage")]
+        public async Task<IActionResult> ServicePage(Guid id, bool draft = false)
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<ServicePage>(id, HttpContext.User, draft);
+
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();
+            }
+        }
+        
+        /// <summary>
+        /// Gets the managed service page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/managedservicespage")]
+        public async Task<IActionResult> ManagedServicesPage(Guid id, bool draft = false)
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<ManagedServicesPage>(id, HttpContext.User, draft);
 
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();
+            }
+        }
+        
+        /// <summary>
+        /// Gets the contacts page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/contactpage")]
+        public async Task<IActionResult> ContactPage(Guid id, bool draft = false)
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<ContactPage>(id, HttpContext.User, draft);
+
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();

```

Comment:
Same as above, this should be a public page, can't understand the exception handle.